Packs/CVE_2021_44228/ReleaseNotes/1_0_2.md

```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### CVE-2021-44228 - Log4j RCE
+- Added more IoCs to the threat hunting tasks.
```

Comment:
you also made a few inputs changes for some tasks no?

Reply:
Old RN. fixed.